crates/stc_ts_type_checker/tests/conformance/controlFlow/controlFlowGenericTypes.stats.rust-debug

```diff
@@ -1,6 +1,6 @@
 Stats {
     required_error: 6,
     matched_error: 2,
-    extra_error: 12,
+    extra_error: 13,
```

Comment:
Seems like a regression?

Reply:
he need exclude_types_using_fact, but he not has name

Reply:
The result is a false type inference.

Reply:
I think that typefact about truty add logic
`NonNullable`